modules/vfs-class-loader/TESTING.md

```diff
@@ -46,12 +46,17 @@ cp ./modules/vfs-class-loader/target/vfs-reloading-classloader-1.0.0-SNAPSHOT.ja
 
 Stop Accumulo if it's running and add the following to the accumulo-env.sh:
 
+**Note:** Make sure the commons-vfs2 jar is also on the classpath as well. This dependency was removed in Accumulo 3.0.0 
+so if using that version or newer you will need to add it. Also, if you are testing using Uno, do not turn on debug logging 
+as this will cause errors on start up as the logging to standard out interferes with the configuration parsing.
+
```

Comment:
Is this specific to testing the classloader?  Is it the test is reading to logs to config something?  As this read now I have the impression that for any testing we cannot set the log level to debug.

Maybe as a seperate PR, but if feasible, this seems like something that should be fixed so that debug can be enabled.

Reply:
Where does it fail?  Looking at ClusterConfigParserTest it looks like the parser it is reading from file(s) so logging and stdout should not apply? Is it this [outputShellVariables](https://github.com/apache/accumulo/blob/6d5e5933734f74bb4a3c607a34df32fd93cb8916/core/src/main/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParser.java#L143) that then gets parsed in testing?